test/menu-bar.test.js

```diff
@@ -228,9 +228,12 @@ describe('overflow button', () => {
   };
 
   beforeEach(async () => {
-    menu = fixtureSync('<vaadin-menu-bar></vaadin-menu-bar>');
+    container = fixtureSync(
+      '<div style="display: flex;"><vaadin-menu-bar style="min-width: 100%"></vaadin-menu-bar></div>'
+    );
+    menu = container.firstChild;
 
-    menu.style.width = '250px';
+    container.style.width = '250px';
```

Comment:
Avoid changes to the existing fixtures/test logic unless it's necessary (to avoid accidental regressions in their purpose). If you need a different fixture for the new tests, you can always create a new one.